File: src/main/java/at/hrechny/predictionsbot/connector/apifootball/model/Fixture.java
package at.hrechny.predictionsbot.connector.apifootball.model;

import lombok.Data;

@Data
public class Fixture {

  private FixtureData fixture;
  private LeagueData league;
  private TeamsData teams;
  private Score goals;
  private ScoreDetail score;

}

File: src/main/kotlin/at/hrechny/predictionsbot/connector/apifootball/model/Fixture.kt
package at.hrechny.predictionsbot.connector.apifootball.model

class Fixture {
    var fixture: FixtureData? = null
    var league: LeagueData? = null
    var teams: TeamsData? = null
    var goals: Score? = null
    var score: ScoreDetail? = null
}
